app/src/main/java/com/example/harvestfresh/fragments/EventsFragment.java

```diff
@@ -0,0 +1,232 @@
+//            }
+//
+//            return false;
+//        }
+//    }
+//
+//    @Override
+//    protected void onResume() {
+//        super.onResume();
+//
+//        displayLocation();
+//
+//        .startLocationUpdatesWithPermissionCheck(this);
+//    }
+//
+//    private void displayLocation() {
+//        if (mCurrentLocation != null) {
+//            //Toast.makeText(this, "GPS location was found!", Toast.LENGTH_SHORT).show();
+//            LatLng latLng = new LatLng(mCurrentLocation.getLatitude(), mCurrentLocation.getLongitude());
+//            CameraUpdate cameraUpdate = CameraUpdateFactory.newLatLngZoom(latLng, 17);
+//            map.animateCamera(cameraUpdate);
+//        } else {
+//            //Toast.makeText(this, "Current location was null, enable GPS on emulator!", Toast.LENGTH_SHORT).show();
+//        }
+//    }
+//
+//    @NeedsPermission({Manifest.permission.ACCESS_FINE_LOCATION, Manifest.permission.ACCESS_COARSE_LOCATION})
+//    protected void startLocationUpdates() {
+//        mLocationRequest = new LocationRequest();
+//        mLocationRequest.setPriority(LocationRequest.PRIORITY_BALANCED_POWER_ACCURACY);
+//        mLocationRequest.setInterval(UPDATE_INTERVAL);
+//        mLocationRequest.setFastestInterval(FASTEST_INTERVAL);
+//
+//        LocationSettingsRequest.Builder builder = new LocationSettingsRequest.Builder();
+//        builder.addLocationRequest(mLocationRequest);
+//        LocationSettingsRequest locationSettingsRequest = builder.build();
+//
+//        SettingsClient settingsClient = LocationServices.getSettingsClient(this);
+//        settingsClient.checkLocationSettings(locationSettingsRequest);
+//        //noinspection MissingPermission
+//        getFusedLocationProviderClient(this).requestLocationUpdates(mLocationRequest, new LocationCallback() {
+//                    @Override
+//                    public void onLocationResult(LocationResult locationResult) {
+//                        onLocationChanged(locationResult.getLastLocation());
+//                    }
+//                },
+//                Looper.myLooper());
+//    }
+//
+//    public void onLocationChanged(Location location) {
+//        // GPS may be turned off
+//        if (location == null) {
+//            return;
+//        }
+//
+//        mCurrentLocation = location;
+//        String msg = "Updated Location: " +
+//                Double.toString(location.getLatitude()) + "," +
+//                Double.toString(location.getLongitude());
+//        //Toast.makeText(this, msg, Toast.LENGTH_SHORT).show();
+//        displayLocation();
+//    }
+//
+//    public void onSaveInstanceState(Bundle savedInstanceState) {
+//        savedInstanceState.putParcelable(KEY_LOCATION, mCurrentLocation);
+//        super.onSaveInstanceState(savedInstanceState);
+//    }
+//
+//    @Override
+//    public void onMapLongClick(@NonNull LatLng latLng) {
+//
+//    }
```

Comment:
Avoid committing commented code.

Reply:
Sorry, I had deleted the commented code in future commits but will keep in mind to not commit commented code.